r2/r2/lib/automoderator.py

```diff
@@ -98,6 +98,10 @@ def replace_placeholders(string, data, matches):
         "{{author}}": data["author"].name,
         "{{body}}": getattr(item, "body", ""),
         "{{subreddit}}": data["subreddit"].name,
+        "{{author_flair_text}}": getattr(data["author"],
```

Comment:
Actually doing the section separately the top is probably unnecessary, I changed my mind about doing some other stuff in there so it's kind of pointless now. Might as well just do it right in here:

```
            "{{author_flair_text}}": data["author"].flair_text(
                data["subreddit"]._id, obey_disabled=True),
            "{{author_flair_css_class}}": data["author"].flair_css_class(
                data["subreddit"]._id, obey_disabled=True),
```